custom/icds_reports/models/util.py

```diff
@@ -14,6 +14,7 @@ class AggregateSQLProfile(models.Model):
     name = models.TextField()
     date = models.DateField(auto_now=True)
     duration = models.PositiveIntegerField()
+    latest_aggregation = models.DateTimeField(null=True)
```

Comment:
This name is very misleading. It is not the time of the latest aggregation, but the when the most recent doc is. Can you change the name to something clearer, so others will be better able to understand this. Maybe something like `last_included_doc` or `last_included_doc_date`, or something better if you think of one.